lib/bolt/executor.rb

```diff
@@ -17,11 +18,12 @@ def initialize(config = Bolt::Config.new, noop = nil, plan_logging = false)
       @plan_logging = plan_logging
     end
 
-    def from_uris(nodes)
-      nodes.map do |node|
-        Bolt::Node.from_uri(node, config: @config)
+    def from_targets(targets)
```

Comment:
This is now only used internally to - essentially - choose a transport for each node that's used in the `on` method.